infrastructure/lambda/api.yaml

```diff
@@ -46,7 +46,7 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/Error"
-      x-amazon-apigateway-request-validator: "Validate body"
+      x-amazon-apigateway-request-validator: "Validate body" ## todo this validator doesn't exist
```

Comment:
if we set this to `Validate both` then the PUT address call fails - @KaraokeStu could you take a look please?